estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
```suggestion
from . import models

```
Always newline character at the end of your files for UNIX compatibility.
Refer to [PEP 8](https://peps.python.org/pep-0008/) for Python conventions.